### ISSUE TITLE
Trade 100% of amount

### ISSUE THREAD

**Issue by dorienh** [dhedge/dhedge-public-bot#2]
Any possibility to not specify the exact amount but instead set the percentage you want to sell/buy? Not too sure how I could know in Tradingview how much my balance is.

**Comment by hochbruj** [comment]
Yes, I could implement that functionality. We are currently very busy with
the new release at dHedge, so it might take some time.
Also, I can protect the calls with an API key you can set in your project,
so only you who knows the key can execute the functions.

Am Mi., 16. Juni 2021 um 15:44 Uhr schrieb Dorien ***@***.***
